### DISCUSSION TITLE
How to configure pgBackRest

### DISCUSSION THREAD

**Discussion by prensgold** [autobase-tech/autobase#229] [Q&A] [answered]
we've setup postgres cluster with your playbook.
But we forgot that configure  with the pgBackRest for backup.
Now application team works on cluster.
how can we configure pgBackRest with your playbook without reinstall the cluster .

**Comment by vitabaks** [comment, marked as answer]
Hi @prensgold 

Use tags https://github.com/vitabaks/postgresql_cluster/blob/master/tags.md

example:

`ansible-playbook deploy_pgcluster.yml --tags pgbackrest`
